shell/server/pre-meteor.js

```diff
@@ -471,6 +490,12 @@ Meteor.startup(() => {
         return;
       }
 
+      if (id.match(/^selftest/)) {
```

Comment:
(We may want to use this sort of pattern for other things in the future, e.g. API endpoints being `api-*`.)